website/src/data/users.tsx

```diff
@@ -363,6 +363,86 @@ const Users: User[] = [
     source: 'https://github.com/Azure-Samples/svc-invoke-dapr-csharp',
     tags: ['dapr','dotnetCsharp','aca']
   },
+  {
+    title: 'Microservices App - Dapr Albums ACA',
+    description:
+      'Create two Dapr-enabled microservices - album viewer (Node.js) and the album API (C#) and have them communicate using Dapr in an Azure Container Apps environment.',
+      preview: require('./images/dapr-albums-diagram.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Paul Yuknewicz',
+    source: 'https://github.com/Azure-Samples/containerapps-dapralbums',
+    tags: ['dapr','dotnetCsharp', 'aca', "nodejs"]
+  },
+  {
+    title: 'Microservices App - Dapr PubSub Java AKS',
+    description:
+      'Demonstrate event-driven communication between microservices by leveraging the pub/sub messaging architecture using Dapr for a Java application running in an AKS cluster.',
+      preview: require('./images/dapr-pubsub-java-aks-diagram.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Mahmut Canga',
+    source: 'https://github.com/Azure-Samples/pubsub-dapr-aks-java',
+    tags: ['dapr','java','aks']
+  },
+  {
+    title: 'Azure Functions - Text Summarization using AI Cognitive Language Service (C#-Isolated)',
+    description:
+      'This sample shows how to take text documents as a input via BlobTrigger, does Text Summarization processing using the AI Congnitive Language service, and then outputs to another text document using BlobOutput binding.',
+      preview: require('./images/test.png'),
```

Comment:
Did you mean to update these image references?

Reply:
These templates do not have images (yet) so I've left them as "test" for now